lib/templates/template.tex

```diff
@@ -1,5 +1,8 @@
 \documentclass[$if(fontsize)$$fontsize$$else$12pt$endif$,$if(lang)$$lang$,$endif$$if(papersize)$$papersize$,$endif$$for(classoption)$$classoption$$sep$,$endfor$]{$documentclass$}
 
+\providecommand{\tightlist}{%
```

Comment:
Can you add a comment as to why this is here? 

Otherwise we'll forget and it's easy for this boilerplate to build up.